### ISSUE TITLE
StrasApp – Pas d’attribution

### ISSUE THREAD

**Issue by le-jun** [osm-fr/attributions#159]
L’Eurométropole de Strasbourg a lancé l’application StrasApp qui comporte notamment une carte. La source des données cartographiques n’est indiquée nulle part. Par comparaison des [captures d’écrans des stores](https://play-lh.googleusercontent.com/Ybzs7WOP8m1REgLWcGX7Eo2VKbojURfo4LFFazHbJBZvsXa5IqrRw0pHBRXFJdlPeQw=w3484-h1762) avec les cartes courantes, OpenStreetMap semble particulièrement y ressembler, notamment avec les cheminements piétons Place de la République, Parc de l’Étoile ou le Cimetière St-Urbain.

J’ai envoyé un mail aujourd’hui même à l’équipe de développement, [strasapp@strasbourg.eu](mailto:strasapp@strasbourg.eu), pour savoir ce qu’il en est

**Comment by le-jun** [comment]
J’ai reçu une réponse, il semblerait qu’ils fassent appel à [OpenMapTiles](https://openmaptiles.org/) de [MapTiler](https://www.maptiler.com/open-source/) pour leurs tuiles ce qui est ou sera bientot mentionné sur leur app. Du coup on se retrouve avec une situation similaire avec le cas MapBox où les utilisateurs verront Mapbox mais pas OpenStreetMap. J’ai pas suivi l’affaire et le fin mot de l’histoire, on est censé monter au créneau ou on laisse couler ?

**Comment by le-jun** [comment]
Effectivement, MapTiler est propre là dessus. Dans leur README il est bien [indiqué](https://github.com/openmaptiles/openmaptiles#license) de quelle manière l’attribution doit être faite sur les outils utilisant leurs tuiles.

**Comment by AntoninCurtit** [comment]
Y'a des signalements ouverts pour des cas où sont crédités Mapbox ou Mapnik mais pas OSM... Donc comme Altonss le dit : Le crédit à OSM est obligatoire, peut importe combien de services ont mis un coup de peinture dessus avant. (Mais libre à eux de créditer ces autres services)

**Comment by cquest** [comment]
@le-jun tu leur a signalé le readme d'openmaptiles qui indique sans ambiguité qu'OSM doit bien être crédité et pas juste openmaptiles ?

**Comment by AntoninCurtit** [comment]
Sidenote : J'ai contacté hier soir l'agence responsable, pour ce signalement ainsi qu'un autre. Déjà 2 réponses.

Ils n'ont pas forcément lu mais en tout cas ça devrait se corriger vite et bien. 🤞

**Comment by le-jun** [comment]
@cquest Affirmatif. Je connais pas l’étiquette à suivre, j’aurais du rendre publiquement nos échanges ?

**Comment by cquest** [comment]
L'échange pas forcément, mais indiquer ici que l'info leur est parvenue.

On sait mieux comme ça où ils en sont de leur connaissance du non respect de l'attribution.

**Comment by le-jun** [comment]
Mise à jour et correction prévue « d’ici une quinzaine de jours ».

**Comment by fnuttens** [comment]
Ça a été corrigé dans la version 1.0.2, ici sur iOS :

![StrasApp](https://user-images.githubusercontent.com/8812165/163053601-ade6bf86-0098-47ae-8afd-51392fc2e172.png)

**Comment by Altonss** [comment]
Super! Il y a un lien cliquable vers https://www.openstreetmap.org/copyright ? Parce qu'il faut soit mettre ce lien, soit préciser clairement que les données sont disponibles sous la licence ODbL...
Sinon ne reste plus qu'à voir pour la version Android :)
En tous cas merci beaucoup pour la réactivité et les améliorations!

**Comment by fnuttens** [comment]
Oui le lien est présent 🙂

**Comment by Altonss** [comment]
> Oui le lien est présent slightly_smiling_face

Parfait! Merci beaucoup :)

**Comment by Altonss** [comment]
Je laisse encore ouvert pour attendre l'info sur Android :)

**Comment by AntoninCurtit** [comment]
Android
- [x] Attribution
- [x] Lien

Allez @Altonss, je te laisse le privilège de clore l'issue. 😉

**Comment by Altonss** [comment]
> D'accord @Altonss, du coup je me laisse le privilège de clore l'issue... ¯_(ツ)_/¯

Je suis désolé la notification a dû passer à la trappe parmi les nombreuses que j'ai reçues ^^ Merci!
